lms/envs/common.py

```diff
@@ -2323,6 +2323,7 @@
 }
 
 # E-Commerce API Configuration
+ECOMMERCE_EXTERNAL_URL_ROOT = None
```

Comment:
cool, makes sense, I would +1 @jimabramson 's original suggestion of calling it the ECOMMERCE_PUBLIC_URL.